packages/catalog-search/package.json

```diff
@@ -6,9 +6,6 @@
     "type": "git",
     "url": "git+https://github.com/edx/frontend-enterprise.git"
   },
-  "browserslist": [
-    "extends @edx/browserslist-config"
-  ],
```

Comment:
This was added prematurely. Removing because migrating to this shared `@edx/browserslist-config` is blocked on Node 16 upgrade.